src/platform/unittest/TlsTest.cpp

```diff
@@ -269,10 +290,47 @@ struct TlsTest : public ::testing::TestWithParam<bool>
                 OnSecConfigCreateComplete));
         ASSERT_NE(nullptr, Pkcs12SecConfig);
 #endif
+
+        QUIC_CREDENTIAL_CONFIG ClientCredConfigCipherSuite = {
+            QUIC_CREDENTIAL_TYPE_NONE,
+            QUIC_CREDENTIAL_FLAG_CLIENT | QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION |
+                QUIC_CREDENTIAL_FLAG_SET_ALLOWED_CIPHER_SUITES,
+            NULL,
+            NULL,
+            NULL,
+            NULL,
+            QUIC_ALLOWED_CIPHER_SUITE_AES_128_GCM_SHA256
+        };
+        VERIFY_QUIC_SUCCESS(
+            CxPlatTlsSecConfigCreate(
+                &ClientCredConfigCipherSuite,
+                CXPLAT_TLS_CREDENTIAL_FLAG_NONE,
+                &TlsContext::TlsClientCallbacks,
+                &ClientSecConfigAes128,
+                OnSecConfigCreateComplete));
+        ASSERT_NE(nullptr, ClientSecConfigAes128);
+
+        ClientCredConfigCipherSuite.AllowedCipherSuites = QUIC_ALLOWED_CIPHER_SUITE_AES_256_GCM_SHA384;
```

Comment:
Should also test negative cases too.

Reply:
Added some test cases below which cover no allowed cipher suites, and also when an unrecognized flag is passed.